docs/release_notes.rst

```diff
@@ -0,0 +1,8 @@
+.. _release_notes:
+
+Release Notes
+==============
+
+.. changelog::
+    :changelog-url: https://your-project.readthedocs.io/en/stable/#changelog
```

Comment:
Why have the changelog here? Is it generated from the github release notes or does this require extra work?

Reply:
This is generated from the GitHub release notes automatically. changelog-url should point to the sphinx docs once they are made available